src/components/screens/forms/form-replies-screen.tsx

```diff
@@ -84,17 +84,16 @@ const FormRepliesScreen: React.FC = () => {
       flex: 1,
       renderHeader: params => {
         return (
-          <AdminFormRepliesScreenStack direction="row">
-            <AdminFormRepliesScreenText sx={{ fontWeight: "bold" }}>{ params.colDef.headerName }</AdminFormRepliesScreenText>
-          </AdminFormRepliesScreenStack>
+          <AdminFormListStack direction="row">
+            <AdminFormTypographyField sx={{ fontWeight: "bold" }}>{ params.colDef.headerName }</AdminFormTypographyField>
```

Comment:
Headers should use localized strings since we have defined strings for them. Since the actual values can differ between forms it could propably be like `strings.getString(``repliesScreen.headers.${params.colDef.headerName}``, locale) ?? column.title`.